src/taco-kits/TacoKitMetadata.json

```diff
@@ -56,6 +57,150 @@
+            "cordova-cli": "5.1.1",
+            "releaseNotesUri": "http://go.microsoft.com/fwlink/?LinkId=623759",
+            "platforms": {
+                "android": {
+                    "version": "4.0.2"
+                },
+                "ios": {
+                    "version": "3.8.0"
+                },
+                "windows": {
+                    "version": "4.0.0"
+                },
+                "wp8": {
+                    "version": "3.8.1"
+                }
+            },
+            "plugins": {
+                "cordova-plugin-console": {
+                    "version": "1.0.1"
+                },
+                "cordova-plugin-device": {
+                    "version": "1.0.1"
+                },
+                "cordova-plugin-file": {
+                    "version": "2.1.0"
+                },
+                "cordova-plugin-inappbrowser": {
+                    "version": "1.0.1"
+                },
+                "cordova-plugin-splashscreen": {
+                    "version": "2.1.0"
+                },
+                "cordova-plugin-network-information": {
+                    "version": "1.0.1"
+                },
+                "cordova-plugin-dialogs": {
+                    "version": "1.1.1"
+                },
+                "cordova-plugin-geolocation": {
+                    "version": "1.0.1"
+                },
+                "cordova-plugin-media": {
+                    "version": "1.0.1"
+                },
+                "cordova-plugin-device-orientation": {
+                    "version": "1.0.1"
+                },
+                "cordova-plugin-battery-status": {
+                    "version": "1.1.0"
+                },
+                "cordova-plugin-camera": {
+                    "version": "1.2.0"
+                },
+                "cordova-plugin-contacts": {
+                    "version": "1.1.0"
+                },
+                "cordova-plugin-device-motion": {
+                    "version": "1.1.1"
+                },
+                "cordova-plugin-file-transfer": {
+                    "version": "1.2.1"
+                },
+                "cordova-plugin-globalization": {
+                    "version": "1.0.1"
+                },
+                "cordova-plugin-media-capture": {
+                    "version": "1.0.1"
+                },
+                "cordova-plugin-vibration": {
+                    "version": "1.2.0"
+                },
+                "cordova-plugin-statusbar": {
+                    "version": "1.0.1"
+                },
+                "cordova-plugin-whitelist": {
+                    "version": "1.1.0"
+                },
+                "cordova-plugin-legacy-whitelist": {
+                    "version": "1.1.0"
+                }
+
+            }
+        },
+        "tacokit@2.1.0": {
+            "cordova-cli": "5.2.0",
+            "releaseNotesUri": "http://go.microsoft.com/fwlink/?LinkId=623759",
```

Comment:
This release notes link will need to be updated surely? To mention that these aren't validated plugins, but point-in-time?